GitVersionCore/Configuration/Wizard/PickBranchingStrategy2Step.cs

```diff
@@ -0,0 +1,39 @@
+namespace GitVersion
+{
+    using System;
+    using System.Collections.Generic;
+
+    public class PickBranchingStrategy2Step : ConfigInitWizardStep
```

Comment:
Maybe rename "BranchingStrategy2" to PickBranchingStrategryGitHubFlow, same for v1